scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -276,6 +276,29 @@ <h5>
     $('select[multiple]').multiselect({
       buttonWidth: '100%'
     });
+
+    /* Changing the Kind drop-down according to Category drop-down */
+    $('select#category').change(function(){
```

Comment:
Nice with the second select changing according to what's selected on the first one. It's working well when you create a new managed variant.

There are a couple of problems however when you are filtering managed variants:
- If you select all category types (SNV, SV, CANCER SNV, CANCER SV) the second select is only showing the oprions for SNV and CANCER SNV:
![image](https://user-images.githubusercontent.com/28093618/121872421-8849ca80-cd05-11eb-9948-c99693bddb96.png)

- Category select in filter modifies both "Kind" selects in filters and in "Add variant" panel, so if you by chance had some previous selection in filters you might end up in this situation in the "Add variant" panel:
![image](https://user-images.githubusercontent.com/28093618/121872861-0d34e400-cd06-11eb-96aa-8bb7c7ede1bc.png)


Suggestion: let's have this function modifying the second select only in "Add variant" panel and  let's leave the filters as they are (all options shown and selectable regardless of the category selected). What do you say @moedarrah ?

Reply:
My bad, I didn't notice that it affected the filter.